index.html

```diff
@@ -248,51 +186,89 @@ <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
+        <ul>
+          <li>
+            Clock speed (depending on AC or DC power) is consistently high
+          </li>
+          <li>
+            Thermals are high but system can handle it
+          </li>
+        </ul>
+        At this point, if you add more work the system may move into critical.
+      </li>
+      <li>
+        <b>Critical</b>: The system is now about to reach its limits, but it hasn’t reached <em>the</em> limit yet.
+        Critical doesn’t mean that the system is being actively throttled, but this state is not sustainable for
+        the long run and might result in throttling if the workload remains the same. This signal is the last call
+        for the web application to lighten its workload.
+      </li>
+    </ul>
+  </p>
+</section>
+
 <section> <h2>Compute Pressure Observer</h2>
 The Compute Pressure Observer API enables developers to understand the utilization characteristics of a CPU.
 
 <section data-dfn-for="ComputePressureObserverCallback">
   <h3>The <a>ComputePressureUpdateCallback</a> callback</h3>
   <pre class="idl">
     callback ComputePressureUpdateCallback = undefined (
-      ComputePressureEntry update,
+      sequence&lt;ComputePressureEntry&gt; entries,
       ComputePressureObserver observer
     );
   </pre>
-  This callback will be invoked when [=CPU=] [=frequency=] and/or [=utilization=] crosses the user
-  set thresholds.
+  This callback will be invoked when the [=pressure state=] changes.
 </section>
 
 <section  data-dfn-for="ComputePressureObserver"> <h2>The <a>ComputePressureObserver</a> object</h2>
   <p>
-    The {{ComputePressureObserver}} can be used to observe changes in the [=CPU=] [=frequency=] and
-      <a>utilization</a>, following user set thresholds.
+    The {{ComputePressureObserver}} can be used to observe changes in the [=pressure states=].
   </p>
-  <p>
-    Providing a list of thresholds effectively separates the resource usage into buckets. For example,
-    the thresholds list `[0.5, 0.75, 0.9]` defines a 4-bucket scheme, where the buckets cover the ranges
-    `[0-0.5[`, `[0.5-0.75[`, `[0.75-0.9[`, and `[0.9-1.0]`.
-  </p>
-  <aside class="note">
-    <p>
-      The user of the API can request an arbitary number of thresholds, but the User Agent may choose to
-      observe only a subset of those requested.
-    </p>
-    <p>
-      It is recommended that user agents allow at most 5 buckets (4 thresholds) for CPU utilization, and
-      2 buckets (1 threshold) for CPU speed.
-    </p>
-  </aside>
   <pre class="idl">
     enum ComputePressureTarget { "cpu" };
 
     [Exposed=Window]
     interface ComputePressureObserver {
       constructor(ComputePressureUpdateCallback callback);
+
       undefined observe(ComputePressureTarget target, optional ComputePressureObserverOptions options = {});
       undefined unobserve(ComputePressureTarget target);
       undefined disconnect();
       sequence&lt;ComputePressureEntry&gt; takeRecords();
```

Comment:
An example featuring `takeRecords()` would be nice as well.

Reply:
I agree, want to file an issue?

Reply:
https://github.com/WICG/compute-pressure/issues/42

Reply:
I ll file the issue!